exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -62,18 +62,19 @@
 ---
 """
 
-import collections
-import logging
-import re
+from collections import deque
 from itertools import chain
+from json import dumps
+from logging import getLogger
+from re import IGNORECASE, UNICODE, compile
```

Comment:
any reason to change all these imports? I find it can be harder to read because I don't know what UNICODE means, i have to go check the import

Reply:
Not really, just being consistent :shrug: 

On the other hand, it keeps the code shorter since modules are not repeated in the code.